### ISSUE TITLE
Dark theme problem

### ISSUE THREAD

**Issue by wxiaoguang** [woodpecker-ci/woodpecker#721]
Woodpecker: {"source": "https://github.com/woodpecker-ci/woodpecker","version": "next-9feb6be9"}

Screenshot:

![image](https://user-images.githubusercontent.com/2114189/151293209-3006f7c3-e766-4c7b-bee0-080965206c90.png)

There might be 2 problems here:

1. The bottom UI is always using dark theme.
2. The texts are hard to read

**Comment by ivictbor** [comment]
Also some other colors contrast in the design are not [WCAG - compatible](https://webaim.org/articles/contrast/), e.g.

![image](https://user-images.githubusercontent.com/1838656/152352705-70ec083a-b9ae-426a-8d69-cdd344e4dce7.png)

Even large text on AA grade fails which is generally means a lot of issues with reading text on different monitors:

![image](https://user-images.githubusercontent.com/1838656/152352608-9265513d-0bfa-4f81-84b7-0ec13e055c36.png)

Just in case if you have design in Figma, it is prossible easily check WCAG criteria with this plugin https://www.figma.com/community/plugin/748533339900865323/Contrast , though currenlty it is easy to find such colors by human eye

**Comment by anbraten** [comment]
We currently do not have a Figma design, but I would absolutely love to have one. It would be very helpful IMO for developers to follow Figma and it would help designers without Vue, code knowledge to improve / suggest UI changes. I have used Figma a bit in the past, but I am really not an expert, so maybe someone else cloud help creating some components and some main views in there. For the colors we use the list of windicss colors from: https://windicss.org/utilities/general/colors.html

Problems related contrast and some design issues in general are somehow known, but I am not an expert on this, so help is absolutely appreciated.

**Comment by 6543** [comment]
should be improved now :)

**Comment by 6543** [comment]
#985
